x/staking/src/keeper/unbonding.rs

```diff
@@ -443,7 +441,7 @@ impl<
     ) -> Result<(), GasStoreErrors> {
         let store = TransactionalContext::kv_store_mut(ctx, &self.store_key);
         let mut store = store.prefix_store_mut(VALIDATOR_QUEUE_KEY);
-        store.delete(&validator_queue_key(&end_time, end_height))?;
+        store.delete(&validator_queue_key(&end_time, end_height))?; //TODO: should we NOT be using the prefix store here - since the validator_queue_key adds the prefix?
```

Comment:
What about app hash? If it correct then no, but in other case we should omit prefix store,

Reply:
I need to check whether we have a test which takes this path. If we do then I'll remove the TODO, if we don't then I'll add a test.